assets/js/atomic/blocks/product-elements/title/block.tsx

```diff
@@ -120,7 +126,8 @@ export const Block = ( props: Props ): JSX.Element => {
 				disabled={ ! showProductLink }
 				name={ product.name }
 				permalink={ product.permalink }
-				rel={ showProductLink ? 'nofollow' : '' }
+				rel={ 'nofollow' + linkRel }
```

Comment:
This could produce invalid value because of a missing space and if linkRel was provided.

Also do we know why we use `nofollow` here? This should be used only for external links afaik that we do not want google to follow. In case of a product title we want google to continue indexing/traversing the page or tracking navigation - right?

It shouldn't be used for internal pages and as per [google recommendations](https://developers.google.com/search/docs/advanced/guidelines/qualify-outbound-links) we should use [robots.txt disallow](https://developers.google.com/search/docs/advanced/robots/robots_txt#disallow) to control what we do not want indexed/tracked for internal pages

The more I'm thinking about this the more I think if we need `rel` on the product title at all. It is always an internal link to a product and not sure if we can have anything meaningful for `rel` and if we should expose it at all.

I think the post title maybe had an option to have a custom/external link set for it? Or maybe a sponsored post, UGC or something like that.

Reply:
I think the issue you mention about a missing space is controlled in [here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6131/files#diff-e7199a70d4d8ad7ef850a7421730ef85eba7104248b9fb83cfcbd4d2e58a0abaR100) so it would not happen, since the space is added.

About the `nofollow`, I kept it because it was the behavior it had before the change, but I think your reasoning makes total sense, we could get rid of it.

Reply:
Ah, it was collapsed and did not notice it - my bad - but hey something good came out of it! :)

I think we should remove it to not overload people with options that do not make much sense.

Reply:
In WC the title, part of the first anchor in the image below, does not have a `rel` at all, only the Add to Cart link has this `rel="nofollow", which makes sense to me. So I think it's ok to remove the `rel` from the title.

<img width="659" alt="Screenshot 2022-03-31 at 11 19 12" src="https://user-images.githubusercontent.com/1628454/161022448-8149dcf0-b0f7-481f-8287-4ed7be281e05.png">